docs/content/en/docs/migrate/metrics-observe/_index.md

```diff
@@ -2,51 +2,333 @@
 title: Migrate Quality Gates
 description: How to translate Quality Gates into Keptn Metrics and evaluations
 weight: 40
-hidechildren: false # this flag hides all sub-pages in the sidebar-multicard.html
 ---
 
 The SLIs and SLOs used for Keptn v1 quality gates can be ported to
-Keptn KeptnMetrics and KeptnEvaluationDefintions.
+appropriate Keptn facilities:
+
+* [Keptn Metrics](../../implementing/evaluatemetrics.md/)
+  allow you to define and view metrics
+  from multiple data sources in your Kubernetes cluster.
+* Use
+  [Keptn Evaluations](../../implementing/evaluations.md)
+  to do a simple evaluation of the metrics data you capture.
+  To implement this, transfer the information from the Keptn v1
+  [sli.yaml](https://keptn.sh/docs/1.0.x/reference/files/sli/)
+  and
+  [slo.yaml](https://keptn.sh/docs/1.0.x/reference/files/slo/)
```

Comment:
sli.yaml is migrated into  multiple AnalysisValueTemplates, slo.yaml is migrated to single KeptnEvaluationDefinition

Reply:
But this section is about evaluations, not analysis, so this is not relevant.